### ISSUE TITLE
Make flipper accessible

### ISSUE THREAD

**Issue by canstudios-nicolaw** [danielstorey/adapt-flipper#8]
It should be possible to complete the flipper with a screen reader

Currently a screen reader will read through all content continuously without flip, this means that it's possible to read all of the content without completing the component. There is no indication to a screen reader user that this component is interactive.

**Comment by canstudios-nicolaw** [comment]
(Happy for this to be assigned to me)
